bcl/inc/bc_onewire_relay.h

```diff
@@ -0,0 +1,62 @@
+#ifndef _BC_ONEWIRE_RELAY_H
+#define _BC_ONEWIRE_RELAY_H
+
+#include <bc_onewire.h>
+
+#define BC_ONEWIRE_RELAY_FAMILY_CODE 0x29
+
+//! @addtogroup bc_onewire_relay bc_onewire_relay
+//! @brief Driver for BigClown 1-wire relay, chipset: DS2408
+//! @{
+
+typedef enum
+{
+    BC_ONEWIRE_RELAY_Q1 = 0,
+    BC_ONEWIRE_RELAY_Q2 = 1,
+    BC_ONEWIRE_RELAY_Q3 = 2,
+    BC_ONEWIRE_RELAY_Q4 = 3,
+    BC_ONEWIRE_RELAY_Q5 = 4,
+    BC_ONEWIRE_RELAY_Q6 = 5,
+    BC_ONEWIRE_RELAY_Q7 = 6,
+    BC_ONEWIRE_RELAY_Q8 = 7
+
+} bc_onewire_relay_q_t;
+
+//! @brief BigClown 1-wire relay instance
+
+typedef struct
+{
+    uint64_t _device_number;
+    bc_gpio_channel_t _channel;
+    uint8_t _state;
+
+} bc_onewire_relay_t;
+
+//! @brief Initialize relay
+//! @param[in] self Instance
+//! @return true On success
+//! @return false On failure
+
+bool bc_onewire_relay_init(bc_onewire_relay_t *self, bc_gpio_channel_t channel, uint64_t device_number);
+
+//! @brief Set relay to specified state
+//! @param[in] self Instance
+//! @param[in] q Q
```

Comment:
I suggest to expand the description as "Relay channel" and change `bc_onewire_relay_q_t` to `bc_onewire_relay_channel_t` ... and have enum fields as `BC_ONEWIRE_RELAY_CHANNEL_Q1` etc. It is just an idea, but I feel this way it is more descriptive. Thanks!